One test fails

### Short description
```
========================================================================================== FAILURES ==========================================================================================
_______________________________________________________________________ testExamples[ RemoteGraphicsView.py - PyQt5 ] ________________________________________________________________________
Qt: Session management error: Authentication Rejected, reason : None of the authentication protocols specified are supported and host-based authentication failed
Qt: Session management error: Authentication Rejected, reason : None of the authentication protocols specified are supported and host-based authentication failed
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()

Failed RemoteGraphicsView Example Test Located in RemoteGraphicsView.py
------------------------------------------------------------------------------------ Captured stdout call ------------------------------------------------------------------------------------
RemoteGraphicsView

Qt: Session management error: Authentication Rejected, reason : None of the authentication protocols specified are supported and host-based authentication failed
Qt: Session management error: Authentication Rejected, reason : None of the authentication protocols specified are supported and host-based authentication failed
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/graphics/py-pyqtgraph/work-py39/pyqtgraph-0.13.1/pyqtgraph/widgets/RemoteGraphicsView.py", line 322, in renderView
    self.shm.resize(size)
SystemError: mmap: resizing not available--no mremap()

================================================================================== short test summary info ===================================================================================
SKIPPED [1] pyqtgraph/examples/test_examples.py:91: Example requires user interaction
SKIPPED [1] pyqtgraph/examples/test_examples.py:91: Test is being problematic on CI machines
SKIPPED [1] pyqtgraph/examples/test_examples.py:91: No need to test with qtconsole not being installed
SKIPPED [1] tests/test_makeARGB.py:4298: CuPy unavailable to test
SKIPPED [1] tests/graphicsItems/test_ImageItem.py:17: CuPy unavailable to test
SKIPPED [1] tests/graphicsItems/test_ImageItem.py:37: CuPy unavailable to test
SKIPPED [1] tests/graphicsItems/ViewBox/test_ViewBox.py:88: Skipping this test until someone has time to fix it.
==================================================================== 1 failed, 369 passed, 7 skipped in 258.34s (0:04:18) ====================================================================
```

### Expected behavior
n/a

### Real behavior
n/a

### Tested environment(s)

 * PyQtGraph version: 0.13.1
 * Qt Python binding: 5.15.6
 * Python version: 3.9
 * NumPy version: 1.23.1
 * Operating system: FreeBSD 13.1 STABLE
 * Installation method: FreeBSD port

--- a/pyqtgraph/widgets/RemoteGraphicsView.py
+++ b/pyqtgraph/widgets/RemoteGraphicsView.py
@@ -313,7 +313,7 @@ class Renderer(GraphicsView):
                     ## it also says (sometimes) 'access is denied' if we try to reuse the tag.
                     self.shmtag = "pyqtgraph_shmem_" + ''.join([chr((random.getrandbits(20)%25) + 97) for i in range(20)])
                     self.shm = mmap.mmap(-1, size, self.shmtag)
-                elif sys.platform == 'darwin':
+                elif sys.platform == 'darwin' or sys.platform.startswith('freebsd'):
                     self.shm.close()
                     fd = self.shmFile.fileno()
                     os.ftruncate(fd, size + 1)

